src/documents/blog/09-msghero-introduction-goals.html.md

```diff
@@ -20,6 +22,10 @@ I moved on to Haxe when I learned I could get HTML5 and CPP builds for (almost)
 
 Flixel is great for me because I no longer have to even think about updating and rendering. Input is taken care of; spritesheets are handled. Life is good.
 
+I'm also helping with a UI library called [HaxeUI](https://github.com/haxeui/haxeui-core) that I'll be talking about in the future. Stay tuned for that one.
+
+![](/images/blog/haxeui.png)
```

Comment:
This is a picture from the Flixel backend? If so, any way to show that, maybe by showing the debugger overlay or something? :)

Reply:
I like the new image. :) I'd have thought it would look consistent with the "official" version though - looks like it's using a different font, a different background color, etc.. Also, no tabs that I can see. Or are those hidden?